
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
  <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
  <meta http-equiv="Content-Style-Type" content="text/css" />
  <meta name="generator" content="pandoc" />
  <meta name="keywords" content="data science, computational biology, bioinformatics, biostatistics, amr, antibiotic resistance, antimicrobial resistance, microbial ecology" />
  <title>aramavilaherrera-cv</title>
  <style type="text/css">code{white-space: pre;}</style>
<!-- fonts -->
        <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css" integrity="sha384-hWVjflwFxL6sNzntih27bfxkr27PmbbK/iSvJ+a4+0owXq79v+lsFkW54bOGbiDQ" crossorigin="anonymous">
        <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
<!-- -->
  <style type="text/css">
  body {
    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
    font-weight: 200;
    max-width: 800px;
    margin: auto;
    padding: 10px 10px 10px 10px;
    background-color: #fdf6e3;
    color: #586e75;
  }
  @media print {
    body {
      background-color: #fff;
    }
  }
  h1 h2 h3 {
    font-weight: 500;
  }
  /* Section bar */
  h3:before {
    content: "";
    display: inline-block;
    margin-right: 1%;
    width: 10%;
    height: 5px;
    background-color: #073642;
    vertical-align: middle;
  }
  h3 {
    page-break-after: avoid;
  }
  strong {
    font-weight: 300;
  }
  dt,
  dd {
    margin-bottom: 3px;
  }
  dt {
    float: left;
    clear: both;
    width: 17%;
    font-weight: 400;
  }
  dd {
    margin-left: 17%;
    overflow: auto;
  }
  @media screen and (max-width: 480px) {
    dd {
      clear: left;
    }
  }
  p {
    margin-top: 0;
    margin-bottom: 7px;
  }
  blockquote {
  /* opacity: 0.5 */
  /* opacity causes low-res rasterization on print */
    text-align: center;
    font-style: italic;
  }
  blockquote p {
    display: inline-block;
    background-color: #eee8d5;
    padding: 1%;
    margin: auto;
  }
  @media print {
    blockquote p {
      background-color: #e6e6e6;
    }
  }
  a {
    text-decoration: none;
    color: #2aa198;
  }
  a:hover,
  a:active {
    background-color: #eee8d5;
    color: #657b83;
    text-decoration: none;
  }
  /* name, contact, summary header */
  .name-and-summary {
    display: block;
    position: -webkit-sticky;
    position: sticky;
    top: 0;
    left: 0;
    right: 0;
    margin: 0;
    padding: 0px 0px 1px 0px;
    background-color: #fdf6e3;
  }
  @media print {
    .name-and-summary {
      background-color: #fff;
    }
  }
  h1.name {
    font-weight: 200;
    display: inline-block;
    padding: 0 0 0 0;
    margin: 0 0 0 0;
  }
  .social-media,
  .email,
  .address,
  .orcid {
    display: inline-block;
    color: #586e75;
    margin-right: 1%;
  }
  .social-media div {
    display: inline-block;
  }
  .name-and-summary hr {
    display: block;
    border: 0;
    border-bottom: 1px solid #586e75;
    margin-bottom: 0;
    margin-top: 0;
  }
  i[class^="fa fa-"]:after {
    content: ": ";
  }
  /* custom sections */
  h3#programming-languages,
  h3#presentations {
    page-break-before: always;
  }
  .cv-col-list > ul {
    margin: 0;
  }
  .cv-columns > ul ul,
  .cv-columns > .cv-col-list {
    -moz-column-count: 4;
    -webkit-column-count: 4;
    column-count: 4;
  }
  @media screen and (max-width: 480px) {
    .cv-columns > ul ul,
    .cv-columns > .cv-col-list {
      -moz-column-count: 2;
      -webkit-column-count: 2;
      column-count: 2;
    }
  }
  .cv-columns ul {
    padding-left: 1em;
    list-style: none;
  }
  
  </style>
</head>
<body>

<!-- custom header -->
<div class="name-and-summary">
            <h1 class="name">Aram Avila-Herrera</h1>
            <div class="contact">
                            <div class="address"><p><a href="https://aavilaherrera.github.io">aavilaherrera.github.io</a></p></div>
                                                            <div class="email">aram.o.gram+bioinformatics@gmail.com</div>
                                    <div class="email">avilaherrera1@llnl.gov</div>
                                        <hr>
            <div class="social-media">
                            <div class="github-username">
                    <!-- <a href="https://github.com/aavilaherrera"><i class="fa fa-github"></i></a> -->
                    <a href="https://github.com/aavilaherrera"><i class="fab fa-github"></i>aavilaherrera</a>
                </div>
                                        <div class="linkedin-username">
                    <!-- <a href="https://linkedin.com/in/aramavilaherrera"><i class="fa fa-linkedin"></i></a> -->
                    <a href="https://linkedin.com/in/aramavilaherrera"><i class="fab fa-linkedin"></i>aramavilaherrera</a>
                </div>
                        </div>
                            <div class="orcid">
                    <a href="https://orcid.org/0000-0003-2357-1757"><i class="ai ai-orcid"></i>https://orcid.org/0000-0003-2357-1757</a>
                </div>
                    </div>
            <div class="summary">
            <blockquote><p>Versatile data scientist with computational biology expertise.</p></blockquote>
        </div>
    </div>
<!-- end custom header -->

<!-- no title
<div id="header">
<h1 class="title">aramavilaherrera-cv</h1>
-->
</div>
<h3 id="academic-history">Academic History</h3>
<dl>
<dt>2016–present</dt>
<dd><p><strong>Lawrence Livermore National Laboratory</strong>, Livermore, CA</p>
<ul>
<li><strong>Postdoctoral Research Staff</strong></li>
</ul>
</dd>
<dt>2015–2016</dt>
<dd><p><strong>Gladstone Institutes</strong>, San Francisco, CA</p>
<ul>
<li><strong>Bioinformatics Fellow</strong></li>
<li>Supervisor: Prof. Katherine Pollard</li>
</ul>
</dd>
<dt>2009–2015</dt>
<dd><p><strong>University of California San Francisco</strong>, San Francisco, CA</p>
<ul>
<li><strong>PhD</strong> in Biological and Medical Informatics</li>
<li>Adviser: Prof. Katherine Pollard</li>
</ul>
</dd>
<dt>2005–2009</dt>
<dd><p><strong>University of California Santa Cruz</strong>, Santa Cruz, CA</p>
<ul>
<li><strong>BS</strong> in Bioinformatics</li>
<li>Advisers: Prof. Kevin Karplus, Prof. Phillip Berman</li>
</ul>
</dd>
</dl>
<h3 id="publications">Publications</h3>
<ul>
<li><p>Ahmed OM, <strong>Avila-Herrera A</strong>, Tun KM, Serpa PH, Peng J, Parthasarathy S, Knapp J-M, Stern DL, Davis GW, Pollard KS, Shah NM. <a href="https://doi.org/10.1016/j.celrep.2019.04.104">Evolution of mechanisms that control mating in <em>Drosophila</em> males</a>. Cell Reports May 2019 27(9):2527-2536.e4</p></li>
<li><p>Wyman SK, <strong>Avila-Herrera A</strong>, Nayfach S, Pollard KS. <a href="https://doi.org/10.1371/journal.pone.0205749">A most wanted list of conserved microbial protein families with no known domains</a>. PLoS ONE October 2018 13(10):e0205749.</p></li>
<li><p>Be N, <strong>Avila-Herrera A</strong>, Allen JE, Singh N, Checinska Sielaff A, Jaing C, Venkateswaran K. <a href="https://doi.org/10.1186/s40168-017-0292-4">Whole metagenome profiles from ISS particulates</a>. BMC Microbiome July 2017 5:81</p></li>
<li><p><strong>Avila-Herrera A</strong>. <em>Statistical evaluation of coevolution methods for predicting inter-protein contacts</em> (Doctoral dissertation). University of California, San Francisco. 2015.</p></li>
<li><p><strong>Avila-Herrera A</strong>, Pollard K. <a href="http://doi.org/10.1186/s12859-015-0677-y">Coevolutionary analyses require phylogenetically deep alignments and better null models to accurately detect inter-protein contacts within and between species</a>. BMC Bioinformatics August 2015 16:268</p></li>
<li><p>Roan NR, Liu H, Usmani SM, Neidleman J, Müller JA, <strong>Avila-Herrera A</strong>, Gawanbacht A, Zirafi O, Chu S, Dong M, Kumar ST, Smith JF, Pollard KS, Fändrich M, Kirchhoff F, Münch J, Witkowska HE, Greene WC. <a href="http://doi.org/10.1128/JVI.00269-14">Liquefaction of semen generates and later degrades a conserved semenogelin peptide that enhances HIV infection</a>. Journal of Virology July 2014 88:13 7221-7234.</p></li>
</ul>
<h3 id="presentations">Presentations</h3>
<ul>
<li><p><strong>Avila-Herrera A</strong>, Be N, Thissen J, Urbaniak C, Karouia F, Smith DJ, Mehta S, Venkateswaran K, Jaing C. <em>Building a Microbial Catalog of the International Space Station</em>. Poster presented at American Society for Gravitational and Space Research. Nov 2018. Washington, DC.</p></li>
<li><p><strong>Avila-Herrera A</strong>, Be N, Thissen J, Urbaniak C, Karouia F, Smith DJ, Mehta S, Venkateswaran K, Jaing C. <em>Building a Microbial Catalog of the International Space Station</em>. Bioinformatics for the Microbiome Symposium. September 2018. Stanford, CA.</p></li>
<li><p><strong>Avila-Herrera A</strong>. <a href="https://data-science.llnl.gov/latest/workshop-2018#Day%201"><em>Search strategies for antimicrobial resistance associated genes</em></a>. LLNL and UC Inaugural Data Science Workshop. Aug 2018. Livermore, CA</p></li>
<li><p><strong>Avila-Herrera A.</strong>, Zemla A, Ames SK, Pearce R, Allen JE. <em>Maximal match weights and sequence motifs for detection of antimicrobial resistance genes in shotgun metagenomic sequences</em>. Poster presented at LLNL Postdoc Symposium. June 2018. Livermore, CA</p></li>
<li><p><strong>Avila-Herrera A</strong>, Be N, Allen JE, Singh N, Checinska Sielaff A, Jaing C, Venkateswaran K. <em>Whole metagenome profiles from ISS particulates</em>. American Society for Gravitational and Space Research. Oct 2017. Seattle, WA</p></li>
<li><p><strong>Avila-Herrera A</strong>, Pollard K. <em>Performance of methods used to identify coevolving residues between proteins</em>. Poster presented at 14th Biomedical Computation at Stanford Symposium. 2014. Stanford, CA</p></li>
<li><p><strong>Avila-Herrera A</strong>, Hernandez R, Krogan N, Pollard K. <em>Measuring correlated evolution in HIV-human protein complexes</em>. Poster presented at XVIth Bay Area Microbial Pathogenesis Symposium. 2013. San Francisco, CA</p></li>
</ul>
<h3 id="research-experience">Research Experience</h3>
<dl>
<dt>2016–present</dt>
<dd><p>I provided bioinformatics and biostatistics support for multiple projects. I used compositional data analysis methods to help characterize the microbial taxonomy of the International Space Station surfaces and crew. We explored machine learning models of complex blast injuries for the purpose of identifying potential medical interventions using patient, treatment, and microbiome measurements taken over time. I also co-developed and benchmarked in-house tools for antimicrobial resistance detection and reporting.</p>
</dd>
<dd><p>Microbes have fascinating relationships with their environments, be they natural or man-made. This includes other microbes and host organisms, such as humans and plants, in addition to their surroundings, like ocean, soil, and buildings. I want to understand how these relationships affect us on Earth and in manned spaceflight.</p>
</dd>
<dt>2015–2016</dt>
<dd><p>In the lab of Prof. Katherine Pollard, I analyzed metagenomic data sets from human and ocean microbiomes for associations between environmental variables and novel gene families of unknown function.</p>
</dd>
<dt>2010–2015</dt>
<dd><p>For my dissertation, advised by Prof. Katherine Pollard, I investigated statistical methods for identifying coevolving residues—the building blocks of proteins. Residues may coevolve in order to conserve important functions or structures between phylogenetically broad protein families, for example, those involved in signaling complexes across bacteria, or in evolutionary arms race between a virus and host.</p>
</dd>
<dt>2009–2010</dt>
<dd><p>I completed three research rotations with Profs. Joseph DeRisi, Alan Frankel, Katherine Pollard.</p>
<ol type="1">
<li>(experimental) Under the supervision of Florencia Caro, I cultured <em>Plasmodium falciparum</em> in blood, coimmunoprecipitated RNA binding proteins and isolated their RNA cargo.</li>
<li>(computational) I implemented two methods for predicting intraprotein evolutionary constraints and applied them to HIV Rev.</li>
<li>(computational) I implemented a metagenomics pipeline for identifying CRISPRs in oceanic samples and tested for associations between their abundance and environmental variables.</li>
</ol>
</dd>
<dd><p>Our PhD training program included two <em>Team Challenges</em>:</p>
<ol type="1">
<li>We conducted microarray experiments and analyses to determine cause of death for yeast samples.</li>
<li>We built a rudimentary fluorescence microscope / spectrometer.</li>
</ol>
</dd>
<dt>Summer 2008</dt>
<dd><p>Minority Access to Research Careers (MARC) Summer Research Institute:</p>
<ul>
<li>Determined the structure of a diketopiperazine sample.</li>
<li>Screened arthropods for the pathogen <em>Wolbachia pipientis</em>.</li>
</ul>
</dd>
<dt>2007–2008</dt>
<dd><p>In the lab of Prof. Phillip Berman, I tested sera for immunoreactivity and automated the digitization of critical biological sequence records.</p>
</dd>
</dl>
<h3 id="teaching-experience">Teaching Experience</h3>
<dl>
<dt>2010–2012 (Falls)</dt>
<dd><p><strong>Integrated Program in Quantitative Biology</strong>, UCSF</p>
<dl>
<dt>Bootcamp Bioinformatics Leader</dt>
<dd><p>Responsible for the organization and design of three-day theoretical and applied bioinformatics crash course. Prepared simulated data sets for coursework and led journal club discussions.</p>
</dd>
<dt>Bootcamp Instructor</dt>
<dd><p>Gave introductory lectures on probability and statistics to students from diverse fields.</p>
</dd>
</dl>
</dd>
<dt>2010 (Fall)</dt>
<dd><p><strong>School of Pharmacy</strong>, UCSF</p>
<dl>
<dt>Assistant in Instuction</dt>
<dd><p>Developed course material, led weekly discussions, held office hours, and administered course website. BPS 112: “Biostatistics”.</p>
</dd>
</dl>
</dd>
</dl>
<h3 id="service">Service</h3>
<dl>
<dt>Referee</dt>
<dd><p><strong>BMC Bioinformatics</strong>, <strong>Bioinformatics</strong>, <strong>PLOS Computational Biology</strong></p>
<dl>
<dt>2017–</dt>
<dd><p>Reviewed research manuscripts for publication.</p>
</dd>
</dl>
</dd>
<dt>Early Career Panelist</dt>
<dd><p><strong>LLNL</strong>; <strong>Gladstone Institutes</strong></p>
<dl>
<dt>Feb 2019; Apr 2017</dt>
<dd><p>Shared early career experiences with undergraduate and graduate students.</p>
</dd>
</dl>
</dd>
<dt>Dec 2014</dt>
<dd><p><strong>Science Education and Outreach</strong>, Gladstone Institutes</p>
<dl>
<dt>Guest Lecturer</dt>
<dd><p>Co-led lesson on biology, prevention, and history of HIV/AIDS to students of Balboa High School, San Francisco.</p>
</dd>
<dd>Co-led hands-on activity demonstrating the importance of HIV testing.
</dd>
</dl>
</dd>
<dt>Sep 2014</dt>
<dd><p><strong>American Heart Association</strong>, San Francisco</p>
<dl>
<dt>Heart Walk Team Leader</dt>
<dd><p>Built a team to fundraise and participate in the 2014 Bay Area Heart Walk.</p>
</dd>
</dl>
</dd>
<dt>2012–2015</dt>
<dd><p><strong>Minority Graduate Student Organization</strong>, UCSF</p>
<dl>
<dt>Peer Mentor</dt>
<dd><p>Provide longterm mentorship for new students.</p>
</dd>
</dl>
</dd>
</dl>
<h3 id="professional-development">Professional Development</h3>
<dl>
<dt>Nov 2018</dt>
<dd><p><strong>National Laboratory Entrepreneurship Academy</strong>, UC Davis GSM</p>
<p>Received interactive entrepreneurship training from experts in academia, government, and industry.</p>
</dd>
<dt>Fall 2013</dt>
<dd><p><strong>CATALYST Award Student Internship Program</strong>, UCSF</p>
<p>Reviewed funding proposals with academic and industry experts in translational research for</p>
<ul>
<li>Unmet need</li>
<li>Commercialization potential</li>
<li>Development feasibility</li>
</ul>
</dd>
<dt>2012</dt>
<dd><p><strong>Graduate Student Internships and Career Exploration</strong>, UCSF</p>
<p>Participated in career exploration and professional development workshops</p>
</dd>
</dl>
<h3 id="honors-and-awards">Honors and Awards</h3>
<dl>
<dt>2015</dt>
<dd>Gladstone Institute of Cardiovascular Disease Award for Excellence in Leadership
</dd>
<dt>2011–2012</dt>
<dd>UCSF Graduate Research Mentorship Fellowship
</dd>
<dt>2011</dt>
<dd>National Science Foundation Graduate Research Fellowship Program (Honorable Mention)
</dd>
<dt>2008–2009</dt>
<dd>Minority Access to Research Careers (MARC) Fellow
</dd>
<dt>2007</dt>
<dd>International Scholar Laureate Program (Delegation on Engineering, China)
</dd>
</dl>
<h3 id="programming-languages">Programming Languages</h3>
<p>(and frequently used libraries)</p>
<div class="cv-columns">
<ul>
<li><strong>Python</strong>
<ul>
<li>jupyter</li>
<li>biopython</li>
<li>networkx/igraph</li>
<li>pandas</li>
<li>BeautifulSoup</li>
<li>numpy/scipy</li>
<li>scikit-learn</li>
</ul></li>
<li><strong>R</strong>
<ul>
<li>rmarkdown</li>
<li><a href="https://www.tidyverse.org/">tidyverse</a></li>
<li>igraph</li>
<li>Biostrings</li>
<li>phyloseq</li>
<li>vegan</li>
<li>plot.ly</li>
</ul></li>
<li><strong>Web/Viz.</strong>
<ul>
<li>Jekyll</li>
<li>JavaScript</li>
<li>CoffeeScript</li>
<li>CSS</li>
<li>Stylus</li>
<li>HTML5</li>
<li>D3.js</li>
</ul></li>
<li><strong>Misc.</strong>
<ul>
<li><strong><em>bash</em></strong></li>
<li><strong><em>Perl</em></strong></li>
<li><strong><em>LaTeX</em></strong></li>
<li><strong>SQL</strong></li>
<li><strong>C</strong></li>
<li>XML</li>
<li>Java</li>
<li>C++</li>
<li>MATLAB</li>
</ul></li>
</ul>
</div>
<h3 id="foreign-languages">Foreign Languages</h3>
<ul>
<li>Spanish (fluent)</li>
<li>French (proficient)</li>
</ul>
</body>
</html>
